tests/Integration/AbstractTechproductsTest.php

```diff
@@ -647,6 +648,104 @@ public function testGetStatusFromAllCoreWhenNoCoreNameWasSet()
         $response = $this->client->coreAdmin($coreAdminQuery);
         $this->assertTrue($response->getWasSuccessful());
     }
+
+    public function testManagedStopwords()
+    {
+        $query = $this->client->createManagedStopwords();
+        $query->setName('english');
+        $term = 'the';
+
+        // Add stopwords
+        $add = new AddStopwords();
+        $add->setStopwords([$term]);
+        $query->setCommand($add);
+        $result = $this->client->execute($query);
+        $this->assertEquals(200, $result->getResponse()->getStatusCode());
+
+        // Check if single stopword exist
+        $exists = new ExistsStopwords();
+        $exists->setTerm($term);
+        $query->setCommand($exists);
+        $result = $this->client->execute($query);
+        $this->assertEquals(200, $result->getResponse()->getStatusCode());
+
+        // List stopwords
+        $query->removeCommand();
```

Comment:
move this line above the comment and add another comment to explain the purpose, even if I already got it ;-)